octo_client/client.py

```diff
@@ -147,21 +149,23 @@ def _make_request(
         return response_json
 
     def _filter_request_log_data(
-            self, data_length: int, content: Union[str, dict]
+        self, data_length: int, request_content: Union[str, dict]
     ) -> Optional[Union[str, dict]]:
         if self.log_requests:
+            content = hide_sensitive_data(copy.deepcopy(request_content))
             if self.log_size_limit and data_length > self.log_size_limit:
                 return "TRUNCATED"
             return content
         return None
 
     def _filter_log_data(
-            self, response_length: int, response_content: Union[str, dict]
+        self, response_length: int, response_content: Union[str, dict]
     ) -> Optional[Union[str, dict]]:
         if self.log_responses:
+            content = hide_sensitive_data(copy.deepcopy(response_content))
             if self.log_size_limit and response_length > self.log_size_limit:
```

Comment:
question: Wouldn't make sense to use the length of `content` instead of the `response_length` to see whether we exceeded the limit?

Reply:
I don't think that there is any difference, since the `response_length` is actually just `len(str(response_length))` being passed as a different argument.

Reply:
I do to think that it will be much improvement, meaning that we will have less TRUNCATED logs, but as the `content`  (and not the `response_content`) is now the log data we want to display, I think it would be more correct to use `len(content)` instead of `response_length` to decide if we need to truncate the response

Reply:
Ahh I do see your point now, I will change to `len(content)`